### PR TITLE
Fix for issue 327: order by relation

### PR DESCRIPTION
**Pull request by rcomblen**
- Solving issue https://github.com/jfinkels/flask-restless/issues/327

### CODE REVIEW

File: tests/test_search.py [hunk 1]
```diff
@@ -90,6 +90,104 @@ def test_query_related_field(self):
...
+        mary = self.session.query(self.Person).filter_by(name=u'Mary').first()
+        mary.computers.append(computer_for_mary)
+        computer_for_lucy = self.Computer(name=u'2nd', vendor=u'vendor')
+        lucy = self.session.query(self.Person).filter_by(name=u'Lucy').first()
+        lucy.computers.append(computer_for_lucy)
+        self.session.commit()
+
+        # When
+        d = {
+            'filters': [{'name': 'owner', 'op': 'has', 'val': {'name': 'name', 'op': 'like', 'val': '%y'}}],
```
Comment (jfinkels) [inline tests/test_search.py]: Trim line to at most 79 characters.

File: tests/test_search.py [hunk 2]
```diff
@@ -90,6 +90,104 @@ def test_query_related_field(self):
...
+        query = create_query(self.session, self.Computer, d)
+
+        # Then
+        assert query.count() == 2
+        results = query.all()
+        assert results[0].owner.name == u'Mary'
+        assert results[1].owner.name == u'Lucy'
+
+
+
```
Comment (jfinkels) [inline tests/test_search.py]: One too many newlines here.

File: tests/test_search.py [hunk 3]
```diff
@@ -90,6 +90,104 @@ def test_query_related_field(self):
         assert results[1].other == 19
```
Comment (jfinkels) [inline tests/test_search.py]: Remove this blank line; there should be only one between functions.

**Comment by rcomblen** [comment]
Thanks for the feedback. If some kind of compliance is mandatory (e.g. pep8 or pylint), it might make sense to describe it in the README.md.

**Comment by jfinkels** [comment]
I'll make a note in the README.

**Comment by jfinkels** [comment]
This looks good, thank you!

**Comment by rcomblen** [comment]
When is this going to be in an official release ?

**Comment by jfinkels** [comment]
I'd like to pull in another change before releasing this as version 0.16.0.

**Pull request merged**